tools/policy_master_permission_generation/validated_policy_templates.yaml

```diff
@@ -279,6 +280,7 @@ validated_policy_templates:
 -  "./tools/catalog_management/policy_sync/policy_sync.pt"
 -  "./tools/catalog_management/hidden_policies/hidden_policies.pt"
 -  "./operational/flexera/cco/billing_centers_from_dimensions/billing_centers_from_dimensions.pt"
+-  "./operational/aws/shared_cost_reallocation/aws_support/shared_cost_reallocation_aws_support.pt"
 # Turbonomic
```

Comment:
```suggestion
-  "./operational/flexera/cco/billing_centers_from_dimensions/billing_centers_from_dimensions.pt"
# Turbonomic
```
I don't think we need this listed under Flexera as it is already listed under AWS